src/unit_tests/wazuh_db/test_wdb_global_parser.c

```diff
@@ -20,6 +20,17 @@ typedef struct test_struct {
     char *output;
 } test_struct_t;
 
+char* group_name_generator(int length) {
```

Comment:
As this is not a test, add a brief comment explaining what this function does.

Reply:
Good point. Added!